OpenRA.Mods.Common/Traits/World/PaletteFromFile.cs

```diff
@@ -18,12 +18,22 @@ class PaletteFromFileInfo : ITraitInfo
 	{
 		[Desc("internal palette name")]
 		public readonly string Name = null;
+
 		[Desc("If defined, load the palette only for this tileset.")]
 		public readonly string Tileset = null;
+
 		[Desc("filename to load")]
 		public readonly string Filename = null;
+
 		[Desc("Map listed indices to shadow. Ignores previous color.")]
 		public readonly int[] ShadowIndex = { };
+
+		[Desc("Converted to transparency.")]
+		public readonly int BackgroundIndex = 0;
+
+		[Desc("Converted to transparency.")]
```

Comment:
This isn't true.
Perhaps something like 'Converts `BackgroundIndex` to transparency.` would be better.